models/modelsmodule.cpp

```diff
@@ -70,6 +70,7 @@
 #include "iaf_psc_exp.h"
 #include "iaf_psc_exp_htum.h"
 #include "iaf_psc_exp_multisynapse.h"
+#include "cm_default.h"
```

Comment:
Maybe leave this in alphabetic order (also the call to ``register_node_model()`` below).

Reply:
done!